packages/cashier/src/Components/Form/__tests__/account-transfer-form.spec.js

```diff
@@ -0,0 +1,451 @@
+        const props = mockProps();
+        props.selected_from.is_mt = true;
+        props.selected_to.is_mt = true;
+        props.transfer_fee = 0;
+
+        render(<AccountTransferForm {...props} is_dxtrade_allowed />);
+
+        expect(
+            screen.getByText(
+                'We do not charge a transfer fee for transfers in the same currency between your Deriv fiat and DMT5 accounts and between your Deriv fiat and Deriv X accounts. Please bear in mind that some transfers may not be possible.'
+            )
+        ).toBeInTheDocument();
+    });
+
+    it('should show proper note if no transfer fee and is_dxtrade_allowed is false ', () => {
+        isMobile.mockReturnValue(true);
+        const props = mockProps();
+        props.selected_from.is_mt = true;
+        props.selected_to.is_mt = true;
+        props.transfer_fee = 0;
+
+        render(<AccountTransferForm {...props} is_dxtrade_allowed={false} />);
+        expect(
+            screen.getByText(
+                'You’ll not be charged a transfer fee for transfers in the same currency between your Deriv fiat and DMT5 accounts. Please bear in mind that some transfers may not be possible.'
+            )
+        ).toBeInTheDocument();
+    });
+
+    it('should show proper note if transfer fee is 1% and is_dxtrade_allowed ', () => {
+        isMobile.mockReturnValue(true);
+        const props = mockProps();
+        props.selected_from.is_mt = true;
+        props.selected_to.is_mt = true;
+        props.transfer_fee = 1;
+
+        render(<AccountTransferForm {...props} is_dxtrade_allowed />);
+
+        expect(
+            screen.getByText(
+                'We’ll charge a 1% transfer fee for transfers in different currencies between your Deriv fiat and DMT5 accounts and between your Deriv fiat and Deriv X accounts. Please bear in mind that some transfers may not be possible.'
+            )
+        ).toBeInTheDocument();
+    });
+
+    it('should show proper note if transfer fee is 1% and is_dxtrade_allowed is false ', () => {
+        isMobile.mockReturnValue(true);
+        const props = mockProps();
+        props.transfer_fee = 1;
+        props.selected_from.is_mt = true;
+        props.selected_to.is_mt = true;
+
+        render(<AccountTransferForm {...props} is_dxtrade_allowed={false} />);
+
+        expect(
+            screen.getByText(
+                'We’ll charge a 1% transfer fee for transfers in different currencies between your Deriv fiat and DMT5 accounts. Please bear in mind that some transfers may not be possible.'
+            )
+        ).toBeInTheDocument();
+    });
+
+    it('should show proper note if transfer fee is 2% and is_crypto_to_crypto_transfer', () => {
+        isMobile.mockReturnValue(true);
+        const props = mockProps();
+        props.selected_from = { currency: 'USD', is_crypto: true };
+        props.selected_to = { currency: 'USD', is_crypto: true };
+        props.transfer_fee = 2;
```

Comment:
could we change currencies to crypto ? for example BTC and ETH, to not be confused of using USD in this test-case